microsetta_private_api/api/_sample.py

```diff
@@ -134,7 +136,16 @@ def update_sample_association(account_id, source_id, sample_id, body,
             sample_datetime = fromisotime(sample_datetime)
         except ValueError:
             raise BadRequest("Invalid sample_datetime")
-
+        n = datetime.now(sample_datetime.tzinfo)
+        delta = relativedelta(year=n.year-10)
+        n = n+delta
+        lower_limit = n+delta
+        n = datetime.now(sample_datetime.tzinfo)
+        delta = relativedelta(month=n.month+1)
+        n = n+delta
+        upper_limit = n+delta
```

Comment:
Let's streamline this code a bit to be more readable:
```suggestion
        curdate = datetime.now(sample_datetime.tzinfo)
        lower_limit = curdate + relativedelta(years=-10)
        upper_limit = curdate + relativedelta(months=+1)
```